src/Agent.Listener/DistributedTask.Pipelines/Yaml/Contracts/Variable.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Microsoft.TeamFoundation.DistributedTask.Orchestration.Server.Pipelines.Yaml.Contracts
+{
+    internal sealed class Variable : IVariable
```

Comment:
I removed the `Verbatim` property which was an idea that was never implemented. The idea was to retain the exact name when propagating to the task child process (rather than convert to uppercase and underscores). I don't think it's required anymore since `env` is a first class concept now on `task`.